scrimage-core/src/main/java/com/sksamuel/scrimage/nio/internal/GifSequenceReader.java

```diff
@@ -700,11 +718,11 @@ protected void readImage() {
 
       // create new image to receive frame data
       image =
-         new BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB_PRE);
+         new BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB);
```

Comment:
I'm not fully sure of the impact this has, so I want to highlight this change for some discussion. The reason I changed it was so I could use the pixels() method in test to ensure that each frame appeared correctly as expected while using the various dispose methods.

Reply:
This was probably a mistake to begin in. I don't think people would expect we use a pre-multiplied alpha.